### ISSUE TITLE
Div id mismatch in docs for modal

### ISSUE THREAD

**Issue by mrbodoia** [twbs/bootstrap#21831]
In the documentation on Modals, there is a Live Demo with the following code
```
<!-- Button trigger modal -->
<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
  Launch demo modal
</button>

<!-- Modal -->
<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        ...
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Save changes</button>
      </div>
    </div>
  </div>
</div>
```

However, copying and running this code doesn't immediately work, because the button has `data-target="#exampleModal"` while the modal div has `id="myModal"`. Changing these two values so that they match would make the documentation more accurate.

**Comment by Quy** [comment]
Already fixed in #21671.

**Comment by mrbodoia** [comment]
Oops, I guess I didn't search the closed issues thoroughly enough.
